code/__HELPERS/unsorted.dm

```diff
@@ -1520,7 +1520,7 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 //orbit() can run without it (swap orbiting for A)
 //but then you can never stop it and that's just silly.
 /atom/movable/var/atom/orbiting = null
-
+/atom/movable/var/cached_transform = null
```

Comment:
Making an entire var on every movable atom just for caching the transform is gross, but we can't safely do `initial(transform)` in this case, because not everything has default `transform` to begin with.